tests/security_features_test.go

```diff
@@ -188,6 +188,10 @@ var _ = Describe("[Serial][sig-compute]SecurityFeatures", Serial, decorators.Sig
 		Context("With selinuxLauncherType defined as virt_launcher.process", func() {
 
 			It("[test_id:4298]qemu process type is virt_launcher.process, when selinuxLauncherType is virt_launcher.process", func() {
+				if checks.HasFeature(virtconfig.DisableCustomSELinuxPolicy) {
```

Comment:
This should be handled by the filter to avoid false skips. See `automation/test.sh`

Reply:
Thanks for the review!
What do you suggest exactly, a new label?